### PR TITLE
fix: image and k8 for recent fixes

### PR DESCRIPTION
**Pull request by yjkimjunior**

### CODE REVIEW

**Comment by yjkimjunior** [comment]
syncing again,,,

File: back/node_watcher/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "start": "ts-node ./src/index.ts"
   },
   "dependencies": {
+    "@polkadot/api": "^1.0.0-beta.18",
```
Comment (amaurymartiny) [inline back/node_watcher/package.json]: Thibaut removed it from this package, so that we only put it in the root one

Comment (yjkimjunior) [reply back/node_watcher/package.json]: yeah but when we dockerize it, the image will have no `@polkadot/api`

Comment (amaurymartiny) [reply back/node_watcher/package.json]: ah right

**Pull request merged**